proximal/prox_fns/prox_fn.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import division
 import abc
 import numpy as np
+import numexpr as ne
```

Comment:
What is numexpr?

Reply:
Numexpr is a direct competitor of the Halide language, especially for element-wise vector computations on CPUs. It supports fast caching and parallel compute out of the box.

I imported the numexpr library because I saw the the comments `... require Numpy to be performant` in the lines below:

https://github.com/comp-imaging/ProxImaL/blob/2804c19e141b637a5fcbf92acdcae81cab4c6d1e/proximal/prox_fns/prox_fn.py#L81-L86

https://github.com/comp-imaging/ProxImaL/blob/2804c19e141b637a5fcbf92acdcae81cab4c6d1e/proximal/prox_fns/prox_fn.py#L212-L217

Reply:
That sounds great! Maybe I'll use this library elsewhere too.